### PR TITLE
added setup.py 

### LINKED ISSUE #2: Create setup script
A setup.py have to be added to the project.

### PR DESCRIPTION
**Pull request by saadmanrafat**
fixes #2

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+from setuptools import setup, find_packages
+
+NAME = 'aftquery'
+DESCRIPTION = 'A command line tools for discovering info about tennis tournaments and players in Belgiium'
```
Comment (andreiboyanov) [inline setup.py]: I don't write lines longer than 80 characters. Please fix it. In general, I'm following quite strictly pep-8 and also the automatic formatting of the **black** tool.

Comment (andreiboyanov) [reply setup.py]:
```suggestion
DESCRIPTION = 'A command line tools for discovering info about '
              'tennis tournaments and players in Belgium'
```

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        'console_scripts': ['aftquery=aftquery.aftsearch:main']
+    },
+    python_requires=">=3.0",
+    include_package_data=True,
+    license='GPL',
+    classifiers=[
+        'License :: OSI Approved :: GNU General Public License v3 (GPLv3)',
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: Implementation :: CPython',
```
Comment (andreiboyanov) [inline setup.py]: I don"t think we are stuck to a Python implementation.

File: aftquery/aftsearch.py [hunk 3]
```diff
@@ -0,0 +1,97 @@
+#coding=utf-8
+
+import re
+import sys
+import urllib
+import argparse
+from pprint import pprint
+from bs4 import BeautifulSoup as bs4
+
+from .parser.aft import clubs as aft_clubs
```
Comment (andreiboyanov) [inline aftquery/aftsearch.py]: I don't think this will work when you run the script from the command line. Did you test it?

Comment (saadmanrafat) [reply aftquery/aftsearch.py]: yeah aftsearch.py works fine on the refactoring that I did.

**Comment by saadmanrafat** [comment]
I will try to make the changes within a few hours. Also I tried to install

**Pull request closed without merging**